odh/overlays/moc/monitoring/dashboards/operate-first/jupyterhub-usage.yaml

```diff
@@ -0,0 +1,919 @@
+          "pluginVersion": "7.1.1",
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "expr": "kube_pod_container_resource_requests_memory_bytes{namespace=\"opf-jupyterhub\", prometheus_replica=\"prometheus-k8s-0\", pod=~\"jupyterhub-nb-$user_id\"}",
+              "interval": "",
+              "legendFormat": "pod memory requests",
+              "refId": "A"
+            },
+            {
+              "expr": "sum(container_memory_working_set_bytes{container!=\"\", namespace=\"opf-jupyterhub\", pod=~\"jupyterhub-nb-$user_id\"})",
+              "hide": false,
+              "interval": "",
+              "legendFormat": "pod memory usage",
+              "refId": "B"
+            },
+            {
+              "expr": "kube_pod_container_resource_limits_memory_bytes{namespace=\"opf-jupyterhub\", prometheus_replica=\"prometheus-k8s-0\", pod=~\"jupyterhub-nb-$user_id\"}",
+              "interval": "",
+              "legendFormat": "pod memory limits",
+              "refId": "C"
+            }
+          ],
+          "thresholds": [],
+          "timeFrom": null,
+          "timeRegions": [],
+          "timeShift": null,
+          "title": "Memory Resources",
+          "tooltip": {
+            "shared": true,
+            "sort": 0,
+            "value_type": "individual"
+          },
+          "type": "graph",
+          "xaxis": {
+            "buckets": null,
+            "mode": "time",
+            "name": null,
+            "show": true,
+            "values": []
+          },
+          "yaxes": [
+            {
+              "format": "decbytes",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
+            },
+            {
+              "format": "short",
+              "label": null,
+              "logBase": 1,
+              "max": null,
+              "min": null,
+              "show": true
+            }
+          ],
+          "yaxis": {
+            "align": false,
+            "alignLevel": null
+          }
+        }
+      ],
+      "schemaVersion": 26,
+      "style": "dark",
+      "tags": [],
+      "templating": {
+        "list": [
+          {
+            "allValue": null,
+            "current": {
+              "selected": true,
+              "text": "aakankshaduggal745-40gmail-2ecom",
+              "value": "aakankshaduggal745-40gmail-2ecom"
+            },
+            "datasource": "opendatahub",
+            "definition": "kubelet_volume_stats_used_bytes{namespace=\"opf-jupyterhub\"}",
+            "hide": 0,
+            "includeAll": false,
+            "label": "User ID",
+            "multi": false,
+            "name": "user_id",
+            "options": [
```

Comment:
you can just leave the options here empty (`"options": [],`) once you set `refresh: 1` below